### DISCUSSION TITLE
Module '"umi"' has no exported member 'useModel'

### DISCUSSION THREAD

**Discussion by 1195002442** [umijs/umi#6568] [Q&A]
Module '"umi"' has no exported member 'useModel'

![image](https://user-images.githubusercontent.com/27669720/118103990-226bd980-b40d-11eb-9e3d-19d74218bffd.png)
umi g tmp执行未解决
已经安装：
   "@umijs/hooks": "^1.7.1",
    "@umijs/plugin-initial-state": "^2.3.0",
    "@umijs/plugin-model": "^2.5.7",

**Comment by xiaohuoni** [comment]
给你一个最简 demo : https://github.com/umijs/umi/pull/6569

点击下载 [plugin-initial-state.zip](https://github.com/umijs/umi/files/6471632/plugin-initial-state.zip)
